moveit_core/collision_detection/include/moveit/collision_detection/collision_matrix.h

```diff
@@ -102,7 +102,7 @@ class AllowedCollisionMatrix
    *  @param name2 name of second element
    *  @param allowed_collision_type The allowed collision type will be filled here */
   bool getEntry(const std::string& name1, const std::string& name2,
-                AllowedCollision::Type& allowed_collision_type) const;
+                AllowedCollision::Type& allowed_collision) const;
```

Comment:
The automatically suggested names are not always satisfactory. I suggest to drop/revert this commit for now and tackle this separately later.

Reply:
this is not clang-tidy's fault, its a bug by the original author who has the wrong name in the .cpp file.

i think we should make the variable names more consistent. but would you be happier if the "longer" or "more descriptive" variable name was kept instead of the one in the .cpp file?

Reply:
I know. clang-tidy cannot guess the "best" or "intended" name. That's why this type of cleanup requires manual revision and that's the reason I suggested to discuss it in a separate PR.